src/tests/java/htsjdk/variant/vcf/VCFHeaderUnitTest.java

```diff
@@ -61,6 +69,19 @@ private VCFHeader createHeader(String headerStr) {
         return header;
     }
 
+    @BeforeClass
+    private void createTemporaryDirectory() {
```

Comment:
we need to move GATK BaseTest utils down so people don't have to do this anymore.  @droazen any objections?